.changes/v2.22.0/627-features.md

```diff
@@ -0,0 +1,4 @@
+* Added `NsxtEdgeGateway` methods `GetDnsConfig` and `UpdateDnsConfig`. [GH-627]
```

Comment:
Nit. I just realized that we don't have dots at the end of [changelog](https://github.com/vmware/go-vcloud-director/blob/main/CHANGELOG.md) entries. I mean:

```suggestion
* Added `NsxtEdgeGateway` methods `GetDnsConfig` and `UpdateDnsConfig` [GH-627]
```

Reply:
Removed, thanks for noticing